packages/http-client-java/eng/scripts/Test-Packages.ps1

```diff
@@ -37,11 +37,23 @@ try {
                 & ./Setup.ps1
                 & ./Spector-Tests.ps1
                 Set-Location $packageRoot
-                Write-Host "Spector tests passed"
             }
             finally {
                 Pop-Location
             }
+
+            $generatorTestDir = Join-Path $packageRoot 'generator/http-client-generator-clientcore-test'
+            Push-Location $generatorTestDir
+            try {
+                & ./Setup.ps1
+                & ./Spector-Tests.ps1
+                Set-Location $packageRoot
```

Comment:
Seems not necessary? We have Push-Location $packageRoot in https://github.com/microsoft/typespec/pull/5437/files#diff-78fccbc236da104d100c90984618d30e3bc075a29feea4e23ae262b046a710c0R20

Reply:
Yeah, thanks, should not be needed.